#! /bin/sh /usr/share/dpatch/dpatch-run
## 02_use_cli_for_wrapper.dpatch by Christopher James Halse Rogers <raof@ubuntu.com>
##
## All lines beginning with `## DP:' are a description of the patch.
## DP: Use /usr/bin/cli for the wrapper script instead of calling mono
## DP: directly.
## DP: For cli policy conformance.

@DPATCH@
diff -urNad gnome-do-0.8.3.1+dfsg~/Do/gnome-do.in gnome-do-0.8.3.1+dfsg/Do/gnome-do.in
--- gnome-do-0.8.3.1+dfsg~/Do/gnome-do.in	2009-12-08 16:21:50.000000000 +1100
+++ gnome-do-0.8.3.1+dfsg/Do/gnome-do.in	2009-12-09 12:01:25.200827672 +1100
@@ -18,8 +18,8 @@
 
 # If Do is not running, run it.
 if pgrep -u "`id -u`" '^gnome-do$' >/dev/null; then
-	mono "$GNOME_DO_EXE" "$@"
+	/usr/bin/cli "$GNOME_DO_EXE" "$@"
 fi
 while [ "$?" -eq "20" ]; do
-	mono "$GNOME_DO_EXE" "$@"
+	/usr/bin/cli "$GNOME_DO_EXE" "$@"
 done
